docs/referece_template_client_libraries.md

```diff
@@ -0,0 +1,56 @@
+# Method name
+
+General description of the API, disclaimers should also be here:
+
+> **Important note:** This API is currently in Beta and is subject to change. Use of these APIs in production applications is not supported.
+
+- What does the API call do?
+- Why should you use this API?
+- When should you use this API?
+
+## Input
+
+| Parameter       | Type | Required or Optional | Description |
+|:---------------|:--------|:--------| :--------|
+| Command  | Type | Required or Optional | Description of the parameter.  |
+| Trytes  | Type | Required or Optional | Description of the parameter.  |
+
+## Output
+
+| Return type     | Description |
+|:---------------|:--------|
+| Type  | Description of the return type |
+
+| Exceptions     | Description |
+|:---------------|:--------|
+| Exception  | Description of the exception |
+
+## (Optional) Related APIs (link to other product documentation)
+
+| API     | Description |
+|:---------------|:--------|
+| [API name link]()  | Description of the API  |
+
+## (Optional) Permissions and authentication
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+| permission type | List of permissions    |
+| permission type | List of permissions    |
+| permission type | List of permissions    |
+
+## Example
+
+Example call formatted in the correct language block.
+
+```java
+
+```
+
+### Result
+
+Result as returned by the API. Formatted in the correct language block.
+
+```java
+
+```
```

Comment:
An optional `See also` section would be nice to have. Perhaps could look like this:
```
## (Optional) See also
-  Text linking to related API [`API name link`]()
```

Maybe `Related APIs` could be merged here.